src/shared/integrity_op.c

```diff
@@ -40,6 +40,7 @@ char * dbsync_check_msg(const char * component, dbsync_msg msg, long id, const c
     cJSON * data = cJSON_CreateObject();
     cJSON_AddItemToObject(root, "data", data);
     cJSON_AddNumberToObject(data, "id", id);
+    cJSON_AddStringToObject(data, "version", "2.0");
```

Comment:
Why don't we use a numeric version? It's much easier to handle when we need to compare.